src/mmw/js/src/draw/utils.js

```diff
@@ -72,6 +72,13 @@ function placeMarker(map, drawOpts) {
     return defer.promise();
 }
 
+function createRwdMarkerIcon(iconName) {
```

Comment:
Ah, good idea. We can probably skip passing in the message as we'll know which one to show based on the name.